.env.example

```diff
@@ -19,3 +19,4 @@ SEGMENT_ANALYTICS_WRITE_KEY=ENTER_KEY_HERE
 SESSION_MAX_AGE_MS=2592000000
 SESSION_SECRET=CHANGEME
 STRIPE_PUBLIC_API_KEY=ENTER_STRIPE_PUBLIC_KEY_HERE
+GOOGLE_ANALYTICS_ID=ENTER_YOUR_GOOGLE_ANALYTICS_ID
```

Comment:
Should we include Google Analytics by default?
I'm fine with it, but I'd like someone else from the core team to answer that.

Reply:
I'd prefer if you didn't and, instead, stuck with Segment which can report to GA anyway. GA is good for websites but not suitable IMO for web apps such as an e-commerce site.